src/nvim/quickfix.c

```diff
@@ -3450,7 +3450,7 @@ static int qf_goto_cwindow(const qf_info_T *qi, bool resize, int sz,
 static int qf_open_new_cwindow(const qf_info_T *qi, int height)
 {
   win_T *oldwin = curwin;
-  const tabpage_T *const prevtab = curtab;
+  const tabpage_T *const old_curtab = curtab;
```

Comment:
this is a hint that a longer name should be used for the global (bigger scope), since short names are more typical and useful for smaller (local) scopes. Even though you're right about `prevwin` as a precedent, I think `lastused_tabpage` is easier to understand and avoids potential conflicts like this (including when we merge upstream Vim patches).

I would even be in favor of renaming `prevwin` to `lastused_win` , in some later PR maybe.

Reply:
Acknowledged, it would be very bad if a change from upstream in this function referred to `prevtab` intending to refer to the local variable defined here and instead ended up referring to the global variable that's being added.  

This name change is now reverted, and the global is named `lastused_tabpage`.